### PR TITLE
Copy ML-Metadata notebooks into official folder

### PR DESCRIPTION
**Pull request by jialuzh**
Copy ML-Metadata notebooks into official folder after launch. We will all keep a copy in unofficial folder for now until this PR is merged and also all our related docs are updated with the new link.

### CODE REVIEW

**Comment by jialuzh** [comment]
@ivanmkc the test finally passed:) Please help review the PR when you get a chance.

File: ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb [hunk 1]
```diff
@@ -0,0 +1,909 @@
+{
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:1]: <p>Can you cleanup created artifacts like whatever this creates?</p><p> </p><pre class="ql-syntax" spellcheck="false">&nbsp; &nbsp; aiplatform.start_run(run=f"auto-mpg-local-run-{i}")
&nbsp; &nbsp; aiplatform.log_params(params)
&nbsp; &nbsp; model, history = train(
&nbsp; &nbsp; &nbsp; &nbsp; normed_train_data,
&nbsp; &nbsp; &nbsp; &nbsp; train_labels,
&nbsp; &nbsp; &nbsp; &nbsp; num_units=params["num_units"],
&nbsp; &nbsp; &nbsp; &nbsp; activation="relu",
&nbsp; &nbsp; &nbsp; &nbsp; epochs=params["epochs"],
&nbsp; &nbsp; &nbsp; &nbsp; dropout_rate=params["dropout_rate"],
&nbsp; &nbsp; )
&nbsp; &nbsp; aiplatform.log_metrics(
&nbsp; &nbsp; &nbsp; &nbsp; {metric: values[-1] for metric, values in history.history.items()}
&nbsp; &nbsp; )
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

File: ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb [hunk 2]
```diff
@@ -0,0 +1,1073 @@
+{
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:1]: <p>Is there a way to clean up whatever <code>log_metrics</code>  creates?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

File: .cloud-build/requirements.txt [hunk 3]
```diff
@@ -1,3 +1,5 @@
 ipython 
 jupyter>=1.0 
-nbconvert>=6.0
\ No newline at end of file
+nbconvert>=6.0
+numpy
```
Comment (ivanmkc) [inline .cloud-build/requirements.txt:4]: LGTM

File: ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb (hunk 2 above)
Comment (ivanmkc) [inline ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #10.    IS_GOOGLE_CLOUD_NOTEBOOK = os.path.exists("/opt/deeplearning/metadata/env_version")
</pre><p>Remove this line as it was defined before.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

File: ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb (hunk 1 above)
Comment (ivanmkc) [inline ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #10.    IS_GOOGLE_CLOUD_NOTEBOOK = os.path.exists("/opt/deeplearning/metadata/env_version")
</pre><p>Remove this line as it was defined previously</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (jialuzh) [reply ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:1]: Discussed offline, right now, we do not have function to clean up specific artifacts after creation. We are working on enable that capability and will add those into the notebook once supported

File: ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb (hunk 2 above)
Comment (jialuzh) [reply ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:1]: Same as above.

Comment (jialuzh) [reply ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-custom-jobs.ipynb:1]: I actually added this line intentionally as the previous definition was before we restart the kernal and would be lost once kernal restarted after installing required package.

File: ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb (hunk 1 above)
Comment (jialuzh) [reply ai-platform-unified/notebooks/official/ml_metadata/sdk-metric-parameter-tracking-for-locally-trained-models.ipynb:1]: same as above.

**Review by ivanmkc — APPROVED** [review]
LGTM

**Pull request merged**